docs/content/src/learn/tutorials/messaging-integrations/messaging-with-kafka/README.md

```diff
@@ -0,0 +1,272 @@
+# Messaging with Kakfa
+
+Kafka is a distributed, partitioned, replicated commit log service. It provides the functionality of a publish-subscribe messaging system with a unique design. Kafka mainly operates based on a topic model. A topic is a category or feed name to which records get published. Topics in Kafka are always multi-subscriber.
+
+This guide walks you through the process of messaging with Apache Kafka using Ballerina language.
+
+- [Messaging with Kafka](#Messaging-with-Kafka)
```

Comment:
Remove the TOC. When we render this in the docs site, we have an automatically generated TOC on the left so we don't need to replicate this.

Reply:
Done